src/main/java/com/minecolonies/coremod/colony/requestsystem/management/manager/StandardRequestManager.java

```diff
@@ -126,9 +82,13 @@
     @NotNull
     private IRetryingRequestResolver retryingResolver = null;
 
+    @NotNull
+    private int version = -1;
+
     public StandardRequestManager(final IColony colony)
     {
         this.colony = colony;
+        setup();
 
         this.playerResolver = getFactoryController().getNewInstance(TypeConstants.PLAYER_REQUEST_RESOLVER, this);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this call from a constructor to the overridable "getFactoryController" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1699)